build_config/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build_config
-version: 0.4.6-dev
+version: 0.4.6
 description: Support for parsing `build.yaml` configuration.
 repository: https://github.com/dart-lang/build/tree/master/build_config
 
```

Comment:
Could we bump `checked_yaml` to `>=1.0.0 <3.0.0` ?

Reply:
I don't see that available yet...

Reply:
It's not! But I wanted to get that out...so I can break a cycle. 😢